packages/flutter/test/material/checkbox_list_tile_test.dart

```diff
@@ -171,18 +174,56 @@ void main() {
       ),
     );
 
-    expect(tester.widget<Checkbox>(find.byType(Checkbox)).value, null);
+    expect(tester.widget<Checkbox>(find.byType(Checkbox)).value, false);
+
+    // tap checkbox when disable tristate
+    await tester.tap(find.byType(Checkbox));
+    await tester.pumpAndSettle();
+    expect(_value, true);
 
     await tester.tap(find.byType(Checkbox));
     await tester.pumpAndSettle();
     expect(_value, false);
 
+    // tap ListTitle when disable tristate
+    await tester.tap(find.byType(ListTile));
+    await tester.pumpAndSettle();
+    expect(_value, true);
+
+    await tester.tap(find.byType(ListTile));
+    await tester.pumpAndSettle();
+    expect(_value, false);
+
+    // Enable tristate
+    setState(()=> _tristate = true);
```

Comment:
It shouldn't be necessary to call setState() here, rebuilding the frame should be enough.

Reply:
Yes, you are correct, it's redundant. I will fix it, thanks:)

Reply:
Done.